### PR TITLE
Home Page HTML/Route

### PR DESCRIPTION
**Pull request by bwormser**
Added home html and route as well as initial css for login/signup modals. Also edited main.handlears if statement

### CODE REVIEW

**Comment by mpolimen** [comment]
In the home route, if you catch an error, it would be a 500 error, so reflect that in the status and the error rendering.

File: src/routes/home.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+
+router.get('/', async (req, res) => {
+  try {
+    res.sendFile(path.resolve('../src/static/home.html'));
+  } catch (e) {
+    res.status(404).render('error-page', { title: "404 Page Not Found", error: true });
```
Comment (mpolimen) [inline src/routes/home.js:9]: Change to 500 status because this would be a server error

Comment (mpolimen) [inline src/routes/home.js:9]: Change to 500 status because it would be our server failing if it errors.

Comment (bwormser) [reply src/routes/home.js]: Ok changed it

**Comment by ryan280** [comment]
The functionality and pop ups look good. One bug is that if you press the log in button, then the sign up button, it doesn't close the log in pop up (and vice versa).

**Pull request merged**